WalletWasabi.Fluent/Global.cs

```diff
@@ -105,6 +107,12 @@ public Global(string dataDir, Config config, UiConfig uiConfig, WalletManager wa
 			TransactionBroadcaster = new TransactionBroadcaster(Network, BitcoinStore, BackendHttpClientFactory, WalletManager);
 
 			RoundStateUpdaterCircuit = new PersonCircuit();
+
+			Cache = new MemoryCache(new MemoryCacheOptions
```

Comment:
The Cache initialized twice: https://github.com/zkSNACKs/WalletWasabi/pull/7076/files#diff-4303621146ae9199ec56bf180ae5dd201adb1806708f5a80e0c10f6305a67132R144

Reply:
Ah, right. Good catch.